test/e2e/framework.go

```diff
@@ -2284,8 +2284,9 @@ func (data *TestData) GetMulticastInterfaces(antreaNamespace string) ([]string,
 	return agentConf.Multicast.MulticastInterfaces, nil
 }
 
-func GetTransportInterface(data *TestData) (string, error) {
-	cmd := fmt.Sprintf("ip -br addr show | grep %s", clusterInfo.nodes[0].ipv4Addr)
+func (data *TestData) GetTransportInterface() (string, error) {
+	// It assumes all Nodes have the same transport interface name.
+	cmd := fmt.Sprintf("ip -br addr show | grep %s", clusterInfo.nodes[0].ip())
 	if testOptions.providerName == "kind" {
```

Comment:
would be good to do the same thing here and switch to `RunCommandFromPod`? maybe you don't mind doing it in the same PR?

Reply:
done